src/web/templates/member/screen-o-death.js

```diff
@@ -1,6 +1,17 @@
 <script type="text/javascript">
   $(document).ready(function() {
    $('a[data-toggle="tab"]:first').trigger("shown.bs.tab");
-
+   $('.btn-delete-monitor').click(function(e){
+       var monitorName = $(e.target).attr('data');
+       if (!window.confirm('Are you sure you want to delete monitor '+ monitorName + '?')) {
```

Comment:
A modal would look more visually appealing to the end user. It also seems
to be the standard method for user input validation these days. Github and
Twitter are two examples that pop out to me of those who use this.

On Tue, Jan 13, 2015 at 8:24 AM, Sanrag Sood <notifications@github.com>
wrote:

> In src/web/templates/member/screen-o-death.js
> <https://github.com/asm-products/cloudroutes-service/pull/190#discussion-diff-22868539>
> :
>
> > @@ -1,6 +1,17 @@
> >  <script type="text/javascript">
> >    $(document).ready(function() {
> >     $('a[data-toggle="tab"]:first').trigger("shown.bs.tab");
> > -
> > +   $('.btn-delete-monitor').click(function(e){
> > +       var monitorName = $(e.target).attr('data');
> > +       if (!window.confirm('Are you sure you want to delete monitor '+ monitorName + '?')) {
>
> Any particular reasons?
> You'd essentially be recreating the behavior of the javascript Confirm
> dialog, which would not be blocking and would require extra work to make it
> accessible and keyboard friendly. Also, more code to maintain.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asm-products/cloudroutes-service/pull/190/files#r22868539>
> .
>